crypto/ec/ecp_nistz256.c

```diff
@@ -1445,6 +1446,151 @@ static int ecp_nistz256_inv_mod_ord(const EC_GROUP *group, BIGNUM *r,
 # define ecp_nistz256_inv_mod_ord NULL
 #endif
 
+static int ecp_nistz256group_full_init(EC_GROUP *group, const unsigned char *params) {
```

Comment:
this code probably is better in ec_internal_test which already includes ec_local

Reply:
I'm not sure what file you are referring to as an alternative place to move it, but the rest of the method function definitions are all in this (probably too big) file

Reply:
test/ec_internal_test.c

Reply:
Yeah, I don't think that's the right place for this: it belongs with the rest of the method definitions here, and isn't part of a test but the core library

Reply:
Sorry.. I am referring to your test code,, Not sure why I added the line here.. Sorry for the confusion. But the comment still applies :)